aws/data_source_aws_organizations_delegated_administrators_test.go

```diff
@@ -0,0 +1,54 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/organizations"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func TestAccDataSourceAwsOrganizationsDelegatedAdministrators_basic(t *testing.T) {
```

Comment:
Typically, the `_basic` test only sets required parameters. `TestAccDataSourceAwsOrganizationsDelegatedAdministrators_basic` should test the case where `service_principal` is not set, and there should be another test that tests when it is set. Both of these tests could actually delegate two services: the `_basic` test would return both, and the specific test would only return one.